code/__DEFINES/flags.dm

```diff
@@ -4,6 +4,11 @@
 #define ALL (~0) //For convenience.
 #define NONE 0
 
+#define ENABLE_BITFIELD(variable, flag)  ((variable) |= (flag))
+#define DISABLE_BITFIELD(variable, flag) ((variable) &= ~(flag))
+#define CHECK_BITFIELD(variable, flag)   ((variable) & (flag))
+#define TOGGLE_BITFIELD(variable, flag)  ((variable) ^= (flag))
```

Comment:
ninjanomnom and AnturK have commented negatively about these macros in the past, quoting obfuscation